tools/build_rules/prelude_bazel

```diff
@@ -1,2 +1,2 @@
 load("@//tools/build_rules:java.bzl", "java_library")
-
+load("@io_bazel_rules_scala//scala:scala.bzl", "scala_library", "scala_test")
```

Comment:
mm... how do you feel about loading this only in BUILD files that need it?

Reply:
Right, that makes sense given that Scala is only a small percentage of the project.

Reply:
This is a bit tricker than I thought. bazel-deps will not generate the load statements needed for scala_library unless they are specified manually in the `buildHeader`. If you do that, they get added to every BUILD file in `thirdparty/jvm`.

Should we add an option to bazel-deps to autogenerate the load statements based on the types of artifacts that are being written to the BUILD file?